src/sentry/integrations/slack/notify_action.py

```diff
@@ -105,9 +109,12 @@ def clean(self):
                     code="invalid",
                     params=params,
                 )
+            except ApiRateLimitedError:
```

Comment:
Form validation errors are raised so that the error message appears beside the slack field.